CriticalMass/MapViewController.swift

```diff
@@ -135,7 +138,38 @@ class MapViewController: UIViewController {
         mapView.showsUserLocation = true
     }
 
-    // GPS Disabled Overlay
+    private func display(locations: [String: Location]) {
```

Comment:
This method now lives in BikeAnnotationController

Reply:
We can remove it an fix the friend related code in a Future PR

Reply:
https://github.com/criticalmaps/criticalmaps-ios/issues/217